deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/OpenApiGeneratorProcessor.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkiverse.openapi.generator.deployment;
+
+import org.jboss.jandex.ClassInfo;
+
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.CombinedIndexBuildItem;
+import io.quarkus.deployment.builditem.FeatureBuildItem;
+
+class OpenApiGeneratorProcessor {
+
+    private static final String FEATURE = "openapi-generator";
+
+    OpenApiGeneratorConfiguration configuration;
+
+    @BuildStep
+    void discoverGeneratedApis(BuildProducer<GeneratedOpenApiRestClientBuildItem> restClients,
+            BuildProducer<GeneratedOpenApiModelBuildItem> models,
+            BuildProducer<FeatureBuildItem> features,
+            CombinedIndexBuildItem index) {
+
+        boolean hasGeneratedFiles = false;
+        for (SpecConfig spec : configuration.specs.values()) {
+            for (ClassInfo classInfo : index.getIndex().getKnownClasses()) {
+                if (classInfo.name().packagePrefix().equals(spec.apiPackage)) {
+                    restClients.produce(new GeneratedOpenApiRestClientBuildItem(classInfo));
+                    hasGeneratedFiles = true;
+                } else if (classInfo.name().packagePrefix().equals(spec.modelPackage)) {
+                    models.produce(new GeneratedOpenApiModelBuildItem(classInfo));
```

Comment:
maybe I'm misreading it, but wouldn't it make slightly more sense to have only one package name defined and created a name for the other by appending `.model`, or just use one package for both types of generated things?
It would simplify the config for users

Reply:
I was trying to mimic the exact configuration of the underlying library, but I agree that having only one property would simplify things for users. Let's do it.